pkg/storage/pebble.go

```diff
@@ -2085,13 +2089,13 @@ func (e *ExceedMaxSizeError) Error() string {
 }
 
 func pebbleExportToSst(
-	ctx context.Context, reader Reader, options ExportOptions, dest io.Writer,
+	ctx context.Context, cs *cluster.Settings, reader Reader, options ExportOptions, dest io.Writer,
 ) (roachpb.BulkOpSummary, MVCCKey, error) {
 	var span *tracing.Span
 	ctx, span = tracing.ChildSpan(ctx, "pebbleExportToSst")
 	_ = ctx // ctx is currently unused, but this new ctx should be used below in the future.
```

Comment:
We can remove this now, since we're actually using `ctx`.

Reply:
Woops. Missed that! Updated.